### ISSUE TITLE
Doesn't work on 1.16

### ISSUE THREAD

**Issue by weathon** [jmc2obj/j-mc-2-obj#135]
Stack trace:
```
java.lang.ArrayIndexOutOfBoundsException: Index 56 out of bounds for length 55
	at org.jmc.Chunk.getBlocks(Chunk.java:278)
	at org.jmc.Chunk.renderImages(Chunk.java:392)
	at org.jmc.gui.ViewChunkLoaderThread$ChunkImager.run(ViewChunkLoaderThread.java:291)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by danijel3** [comment]
Line in question: https://github.com/jmc2obj/j-mc-2-obj/blob/78db8562141f41df6ad2260cde31fd1c8389f398/src/org/jmc/Chunk.java#L278

**Comment by mmdanggg2** [comment]
Fixed with a06f034adc3130bcb0649be103ceeed28ebf2653
